pkg/webhook/controlplane/ensurer.go

```diff
@@ -234,6 +241,11 @@ func ensureKubeControllerManagerCommandLineArgs(c *corev1.Container, k8sVersion
 
 	c.Command = extensionswebhook.EnsureNoStringWithPrefix(c.Command, "--cloud-config=")
 	c.Command = extensionswebhook.EnsureNoStringWithPrefix(c.Command, "--external-cloud-volume-plugin=")
+
+	// provider-aws is already deploying CCM with --allocate-node-cidrs. We should safely disable IPAM management from KCM.
```

Comment:
provider-aws is already deploying CCM with --allocate-node-cidrs=false.

Reply:
The comment was from another solution that did not work so well. Thank you for noticing